docs/csharp/language-reference/builtin-types/value-types.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Value types - C# reference"
+ms.date: 01/22/2020
+f1_keywords: 
+  - "cs.valuetypes"
+helpviewer_keywords: 
+  - "value types [C#]"
+  - "types [C#], value types"
+  - "C# language, value types"
+ms.assetid: 471eb994-2958-49d5-a6be-19b4313f80a3
+---
+# Value types (C# reference)
+
+*Value types* and [reference types](../keywords/reference-types.md) are the two main categories of C# types. A variable of a value type contains an instance of the type. This differs from a variable of a reference type, which contains a reference to an instance of the type. By default, on [assignment](../operators/assignment-operator.md), passing an argument to a method, or returning a method result, variable values are copied. In the case of value-type variables, the corresponding type instances are copied. The following example demonstrates that behavior:
+
+[!code-csharp[copy of values](~/samples/csharp/language-reference/builtin-types/ValueTypes.cs#ValueTypeCopied)]
```

Comment:
Should the new code reference syntax be used ?

https://docs.microsoft.com/en-us/contribute/how-to-write-use-markdown#snippet-syntax-reference

```suggestion
:::code language="csharp" source="~/samples/csharp/language-reference/builtin-types/ValueTypes.cs" id="ValueTypeCopied":::
```